src/opnsense/mvc/app/controllers/OPNsense/Kea/forms/dialogSubnet4.xml

```diff
@@ -74,6 +74,12 @@
         <allownew>true</allownew>
         <help>Specifies a list of RFC 868 time servers available to the client.</help>
     </field>
+    <field>
+        <id>subnet4.next_server</id>
+        <label>Next server</label>
+        <type>text</type>
+        <help>Next server address or fqdn</help>
```

Comment:
are we sure an fqdn is valid here? https://kea.readthedocs.io/en/kea-2.2.0/arm/dhcp4-srv.html#setting-fixed-fields-in-classification suggests an address.